index.html

```diff
@@ -289,6 +289,124 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <section class="contact-section">
+        <div class="contact-section-container">
+          <div class="contact-section-head">
+            <p class="contact-section-info">
+              I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+          </div>
+          <form class="contact-form" 
+            action="#" 
+            method="POST"
+          >
+            <fieldset class="fieldset">
+              <ul>
+                <li>
+                  <input
+                    type="text"
+                    name="name"
+                    maxlength="30"
+                    id="name"
+                    placeholder="Full name"
+                  
+                  />
+                </li>
+                <li>
+                  <input
+                    type="text"
+                    name="email"
+                    id="email"
+                    placeholder="Email address"
+                   
+                  />
+                </li>
+                
+                <li>
+                  <textarea
+                    id="message"
+                    class="textarea"
+                    cols="30"
+                    rows="10"
+                    maxlength="50"
+                    placeholder="Enter text here"
+                  ></textarea>
+                </li>
+                <li>
+                  <button type="button" class="submit-contacte" id="btn-submit-forme">
+                    Get in touch
+                  </button>
```

Comment:
- [ ] Kindly, make sure our form `button` has a valid `type` attribute in order to be able to submit.
     Check this: `type="submit"`
- [ ] I think it would be better if u stick to the provided design while styling this `button`.
     
    - Your design
![image](https://user-images.githubusercontent.com/87197412/184037840-c5968f63-d46d-4e38-98b2-1aec34331b4a.png)

   - Figma Design 
![image](https://user-images.githubusercontent.com/87197412/184037967-8ff2fe0f-768d-4f22-a042-3cd583c29088.png)

- [ ] Kindly, make sure to implement the button interaction (enable, hover) and style it as requested.

   - Figma Design
  
![image](https://user-images.githubusercontent.com/87197412/184040398-48ea4580-a3af-4bca-b98e-d30182439a1d.png)